Exec/RegTests/Spray-EB/example.inp

```diff
@@ -0,0 +1,126 @@
+# ------------------  INPUTS TO MAIN PROGRAM  -------------------
```

Comment:
@ldowen this case (PeleMP equivalent is EB-test) segfaults. Do you know why that would be?

Reply:
Thank you for fixing this so quickly!